database/migrations/20170531173912_organizations.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = knex =>
+  knex.schema.createTable( 'organizations', table => {
+    table.increments( 'id' ).primary()
+    table.integer( 'user_id' ).notNullable()
```

Comment:
also, this isn't a foreign key. From the user story is looked like an organization would only have one owner...so it's not a many to many association and also maybe at some point a user/owner may delete their account and want to give ownership to another person? Maybe I'm wrong, but if we made it a foreign key then if the user deletes their account the organization would get deleted too? Am I right? @tannerwelsh ?